apps/research-service/src/site-polygons/site-polygons.controller.ts

```diff
@@ -94,7 +97,7 @@ export class SitePolygonsController {
           indicators,
           establishmentTreeSpecies,
           reportingPeriods,
-          siteName
+          sitePolygon.site?.name
```

Comment:
:art: You can leave as-is, but I'm pretty sure this `?` isn't needed because site gets an "required" inner join, meaning that a site polygon won't be returned if the site for it isn't found.